src/numeric/decision_rules.jl

```diff
@@ -17,190 +51,169 @@ function filter_mcoeffs(a::Array{Float64,1},b::Array{Float64,1},n::Array{Int64,1
     return coeffs
 end
 
-# Constant Decision Rule
-type ConstantDecisionRule <: AbstractDecisionRule
-    values::Array{Float64,1}
-end
-(dr::ConstantDecisionRule)(x::Array{Float64,1}) = dr.values
-(dr::ConstantDecisionRule)(x::Array{Float64,2}) = repmat(dr.values', size(x, 1), 1)
-(dr::ConstantDecisionRule)(i::Int, x::Union{Vector,Matrix}) =  dr(x)
-(dr::ConstantDecisionRule)(i::Int, j::Int, x::Union{Vector, Matrix}) = dr(x)
 
 
-## Cubic spline decision rules
+#####
+##### 1-argument decision rule
+#####
 
-function set_values!(dr::AbtsractCubicSplineDR, values::Array{Array{Float64,2},1})
-    grid = dr.grid
-    a = grid.min
-    b = grid.max
-    orders = grid.n
-    dr.coefficients = [filter_mcoeffs(a, b, orders, v) for v in values]
-end
+using splines
```

Comment:
sure that will come. But I'm not a big fan of doing it early as it makes it harder to find the dependence of a particular piece of code, as opposed to the library as a whole.